src/tools/normalize.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Normalize boolean like values to boolean type.
+ */
+export const normalizeBoolean = (value: unknown): boolean | undefined => {
+  const map = new Map<unknown, boolean>()
+  map.set(true, true)
+  map.set(false, false)
+  map.set("true", true)
+  map.set("false", false)
+  return map.get(value)
+}
+
+/**
+ * Windows uses `\` for slash, Unix uses `/` :'(
+ *
+ * Normalize path to use `/` as path separator.
+ */
+export const normalizePath = (input: string) => input.replace(/\\/g, "/")
+
+export const normalize = {
+  boolean: normalizeBoolean,
+  path: normalizePath,
```

Comment:
`normalizePath` was part of some code after I was trying to get the test suites to pass in Windows. Many of the asserts weren't working because directory slashes are different on Windows vs Unix. 

Eventually I abandoned doing this on windows, in favor of WSL Ubuntu and now MacOS. I can remove this utility